projects/plugins/crm/sass/ZeroBSCRM.admin.global.scss

```
@@ -10,12 +10,12 @@
 // Mike's also made a clone of the following in \master\sass\ZeroBSCRM.admin.invoicepreview.scss
 // ... which could be updated if / when needs to be, (to largely match this... won't need all)
 
+// Import Color Studio
+@use '../node_modules/@automattic/color-studio/dist/color-variables' as *;
```

Does it work to just
```suggestion
@use '@automattic/color-studio/dist/color-variables' as *;
```
?

I think with sass-loader it's supposed to.

---

Heh, now I know. I've updated the two instances here and one instance in Jetpack: ead4b3b8ed